feature-libs/order/components/order-details/order-details.module.ts

```diff
@@ -35,9 +44,43 @@ import { OrderDetailItemsComponent } from './order-detail-items/order-detail-ite
 import { OrderDetailReorderComponent } from './order-detail-reorder/order-detail-reorder.component';
 import { ReorderDialogComponent } from './order-detail-reorder/reorder-dialog/reorder-dialog.component';
 import { OrderDetailTotalsComponent } from './order-detail-totals/order-detail-totals.component';
+import {
+  ConsignmentTrackingLinkComponent,
+  DownloadOrderInvoicesDialogModule,
+  OrderDetailsActionsExtendedComponent,
+} from './order-details-extended';
 import { OrderOverviewComponent } from './order-overview/order-overview.component';
 import { defaultReorderLayoutConfig } from './reoder-layout.config';
 
+function registerOrderOutletFactory(
+  token: boolean,
+  outletService: OutletService,
+  componentFactoryResolver: ComponentFactoryResolver
+): () => void {
+  return () => {
+    const config: ProvideOutletOptions = {
+      component: ConsignmentTrackingLinkComponent,
+      id: OrderOutlets.ORDER_CONSIGNMENT,
+      position: OutletPosition.REPLACE,
+    };
+    if (token) {
+      const template = componentFactoryResolver.resolveComponentFactory(
+        config.component
+      );
+      outletService.add(config.id, template, config.position);
+    }
+  };
+}
+
+const enhancedUICmsMapping: CmsConfig = {
+  cmsComponents: {
+    AccountOrderDetailsActionsComponent: {
+      component: OrderDetailsActionsExtendedComponent,
+      guards: [AuthGuard],
```

Comment:
[QUESTION]
Is declaring guards here _really_ needed? I guess the cms-guards declaration is "inherited" from the original component's cms mapping ("inherited" - in terms of deep-merging all the config chunks together).

Note: same question about (re-)declaring guards, applies also to other files in this PR

Reply:
yes, guards are inherited. will do the change